test/integration/features/stories/restart.feature

```diff
@@ -0,0 +1,88 @@
+    When executing "minishift console --url" succeeds
+     And stdout is valid URL
+    Then with up to "40" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+     And "body" of HTTP request to "/console" of OpenShift instance contains "<title>OpenShift Web Console</title>"
+
+  Scenario: User tries oc
+  as she is more like a command line person.
+  (oc environment was set automatically by the test suite here)
+    When executing "oc status" succeeds
+    Then stdout should contain
+    """
+    You have no services, deployment configs, or build configs.
+    Run 'oc new-app' to create an application.
+    """
+
+  Scenario: User deploys example Django application
+  to see if and how Django works on OpenShift.
+    When executing "oc new-app https://github.com/sclorg/django-ex" succeeds
+     And service "django-ex" rollout successfully within "1200" seconds
+     And executing "oc expose svc django-ex" succeeds
+    Then with up to "5" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "django-ex" in namespace "myproject" is equal to "200"
+
+  Scenario: User scales the application
+    When executing "oc scale --replicas=5 dc django-ex"
+    Then exitcode should equal "0"
+
+  Scenario: User tries to deploy the application on Python 2.7
+    When executing "oc patch bc django-ex --patch '{"spec": {"strategy": {"sourceStrategy": {"from": {"name": "python:2.7"}}}}}'" succeeds
+    Then exitcode should equal "0"
+
+  Scenario: User looks at OpenShift logs
+    When executing "minishift logs" succeeds
+    Then stdout should not be empty
+
+  Scenario: User looks at OpenShift config
+    When executing "minishift openshift config view" succeeds
+    Then stdout should be valid YAML
+
+  Scenario: User stops Minishift
+  to take a break and do some other work.
+  Image caching maybe did not finish, but that does not affect Minishift or the cluster at all. 
+   Given Minishift has state "Running"
+    When executing "minishift stop" succeeds
+     And Minishift has state "Stopped"
+
+  Scenario: User starts Minishift again
+  few hours later to continue her explorations.
+  She does not remember all the flags she used before, luckily they are stored in persistent config.
+    When executing "minishift start" succeeds
+    Then Minishift should have state "Running"
+
+  Scenario: User access OpenShift console again
+   Given with up to "20" retries with wait period of "3" seconds container name "k8s_webconsole_webconsole" should be "running"
+    When executing "minishift console --url" succeeds
+     And stdout is valid URL
+    Then with up to "40" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+     And "body" of HTTP request to "/console" of OpenShift instance contains "<title>OpenShift Web Console</title>"
```

Comment:
@agajdosi same here if `/console` retrun 200-OK then we shouldn't be care about the content from the web page. 

Because content might change but `/console` returning the 200 is something will be there.